src/plugins/base64.js

```diff
@@ -0,0 +1,29 @@
+const encodeBase64 = rawSVG => Buffer.from(rawSVG).toString('base64')
```

Comment:
I wonder if we should remove this completely. Base64 is just bad for compression. See data here:

https://github.com/gatsbyjs/gatsby/pull/4205#issuecomment-368348758

Solution: https://www.npmjs.com/package/mini-svg-data-uri

Reply:
@technopagan had something about that I can agree on that <3 also the wrapping we do, not sure if it is useful at all.

Reply:
Actually I expected the CLI to spit out a SVG file, not an Image tag 🙈 

We could keep base64 as plugin and use the mini data uri's as default for encoding.